JeMPI_Apps/JeMPI_API/src/main/resources/application.conf

```
@@ -61,6 +61,19 @@ akka-http-cors {
 
 akka.http.session {
   server-secret = ${JEMPI_SESSION_SECRET}
+  cookie {
+    secure = ${JEMPI_SESSION_SECURE}
+    httpOnly = true
```

```suggestion
    httpOnly = true
```
I'm not sure about this one.

---

removed it